src/game_merge.h

```diff
@@ -167,7 +167,7 @@ struct GameAdd {
     /** The creature model used for determining amount of sacrifices which decrease digger cost. */
     ThingModel cheaper_diggers_sacrifice_model;
     char quick_messages[QUICK_MESSAGES_COUNT][MESSAGE_TEXT_LEN];
-    struct GuiMessage messages[GUI_MESSAGES_COUNT];
+    struct GuiMessageExt messages[GUI_MESSAGES_COUNT];
```

Comment:
You can keep this as GuiMessage I think. Then if it's in the game structure too, rename that one to `GuiMessage_UNUSED`, and make sure all functions use the one from gameadd.